.eslintrc

```diff
@@ -5,6 +5,11 @@
   "parserOptions": {
     "ecmaVersion": 8,
   },
+  "overrides": [{
+    // We exclude test files from dependency injection format checks because
```

Comment:
What do you mean by dependency injection format checks?

Reply:
Following from the Oppia web
https://github.com/oppia/oppia/blob/develop/.eslintrc#L17

Reply:
Hmmm...
If so, I don't think we need the `overrides` section.

Reply:
Done